test_live_unified_chat.py

```
@@ -0,0 +1,227 @@
+"""
+
+import subprocess
+import json
+import time
+import sys
+
+
+def run_curl_command(command):
+    """Run a curl command and return the result."""
+    try:
+        result = subprocess.run(command, shell=True, capture_output=True, text=True)
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)